app/helpers/application_helper.rb

```diff
@@ -34,4 +34,8 @@ def access_denied
     redirect_back(fallback_location: root_path)
     flash[:alert] = 'Доступ заборонено'
   end
+
+  def first_attached_image(issue)
```

Comment:
if this is related to issue only move name please rename it or move to Issue class